lib/db/mem.js

```diff
@@ -847,6 +849,32 @@ module.exports = function (log, error) {
     return P.resolve({})
   }
 
+  Memory.prototype.securityEvent = function (data) {
+    var event = {
+      createdAt: Date.now(),
+      ipAddr: data.ipAddr,
+      name: data.name,
+      uid: data.uid
+    }
+
+    var events = securityEvents[event.uid] || (securityEvents[event.uid] = [])
```

Comment:
Isn't `event.uid` a buffer here? Shouldn't we `toString` it before using it as a key?